npc/common/src/main/java/org/screamingsandals/lib/npc/NPCImpl.java

```diff
@@ -60,8 +65,27 @@
     private volatile boolean lookAtPlayer;
     private final List<MetadataItem> metadata;
 
+    
     private SClientboundSetPlayerTeamPacket.CollisionRule collisionRule;
-
+    private static boolean IS_BUNGEECORD_ENABLED;
+    static {
+        File spigotYmlFile = new File("spigot.yml");
```

Comment:
move this static initializer to NPCManager constructor, add `Path dataFolder` argument to the constructor and make the spigot.yml acquiring relative to the plugin folder: `dataFolder.getParent().resolveSibling("spigot.yml")`